index.html

```diff
@@ -10,21 +10,21 @@
     <h2>Jogo da Velha</h2>
     <table>
         <tr> 
-            <td><div id='pos6' onclick="clic(6)">7</div></td>
-            <td><div id='pos7' onclick="clic(7)">8</div></td>
-            <td><div id='pos8' onclick="clic(8)">9</div></td>
+            <td><div id='pos20' onclick="clic(2,0)">2,0</div></td>
```

Comment:
The main idea is correct. The only wrong thing is that you ordered your rows from bottom to up, and the right approach is ordering it from top to bottom.

So, where you have `2,0` should be `0,0` (line 0, column 0). Where you have `0,2` should be `2,2` (line 2, column 2).

![whatsapp image 2018-07-09 at 12 07 38](https://user-images.githubusercontent.com/11931916/42458848-fc138302-8370-11e8-968b-e20710ded836.jpeg)